search/helpers.py

```diff
@@ -22,7 +22,7 @@ def keyword_in_search(search_item, keywords=()):
 
         if any(
             # fuzzy search
-            fuzz.WRatio(keyword.lower(), information.lower()) > 90 for keyword in keywords
+            fuzz.WRatio(keyword, information, score_cutoff=90) for keyword in keywords
```

Comment:
WRatio both with rapidfuzz and FuzzyWuzzy already lowercases the strings anyways. Using score_cutoff it will return 0 when it is below this score, but can calculate the ratio quicker